examples/neuralNetwork-sequence-hand-gesture-load-model/index.html

```diff
@@ -0,0 +1,28 @@
+<!--
+  👋 Hello! This is an ml5.js example made and shared with ❤️.
+  Learn more about the ml5.js project: https://ml5js.org/
+  ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ 
+  This example demonstrates loading a hand gesture classifier through
+  ml5.neuralNetwork with the sequenceClassificationWithCNN task.
+  This example has been trained with the ASL gestures for Hello and Goodbye.
+-->
+
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ml5.js neuralNetwork Hand Gesture Loading Pre-trained Model Example</title>
```

Comment:
@mop9047 I slightly reworked the last example - please have a look if anything is amiss 🥳